pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -762,7 +776,7 @@ func (c *Cluster) getActivateOSDInitContainer(configDir, namespace, osdID string
 		Command: []string{
 			"/bin/bash",
 			"-c",
-			fmt.Sprintf(activateOSDOnNodeCode, osdInfo.UUID, osdStore, osdInfo.CVMode, blockPathVarName),
+			fmt.Sprintf(activateOSDOnNodeCode, c.clusterInfo.FSID, osdInfo.UUID, osdStore, osdInfo.CVMode, blockPathVarName),
```

Comment:
Perhaps this is the fix for #10375? This must also affect v16.2.9, not just v17.

Reply:
> Perhaps this is the fix for #10375? This must also affect v16.2.9, not just v17.

Indeed. I tested with Ceph v16..2.7 but I've already confirmed master+this PR worked with Ceph v16.2.7. 

> At least temporarily, it would be good to run the pacific test suites against 16.2.7 by changing the pacificTestImage to v16.2.7.

Thanks, I'll do it after revising this PR.